src/streams/UnionStream.js

```diff
@@ -0,0 +1,29 @@
+import { Writable } from 'stream';
+
+class UnionStream extends Writable {
+    /**
+    * copied from: https://www.loggly.com/blog/node-js-libraries-make-sophisticated-logging-simpler/
+    * Create a new instance of MorganLogglyLogger
+    * @param options {Object}
+    * @param options.streams {List[Stream]} list of streams to union
+    */
+    constructor(options) {
+        // allows use to use any JS object instead of only string or Buffer
+        super({ objectMode: true });
+
+        this.streams = options.streams;
+
+        // make sure write function has proper context (this)
+        this._write = this._write.bind(this); // eslint-disable-line
+    }
+
+    _write(message, encoding, callback) {
+        if (typeof message === 'object') {
+            this.streams.map(stream => stream.write(JSON.stringify(message), encoding, callback));
```

Comment:
I have to unravel the `json.stringify/parse` chaining that I'm doing here. I think what I'm doing is overly complicated.

Reply:
Before killing the stream implemntation altogether I want to change the `stream.write` calls here to `stream._write` which might fix my issues with the callback passing.